packages/component-library/stories/lineChart.notes.md

```diff
@@ -0,0 +1,41 @@
+# LineChart Component
+
+## Standard
+
+The Standard story shows the standard usage of the LineChart for the CIVIC platform. LineChart includes standard styling.
+
+These properties can be set in the Standard story:
+
+- Title
```

Comment:
It might be helpful for the developer to note the data types required for each prop.

Reply:
Agreed! If people want to add them, I'm not opposed, but we're thinking that once Storybook Docs launches we'll be able to document prop types in a more automated fashion. It looks like that feature may be slated for Storybook 5.2
https://github.com/storybookjs/storybook/issues/6640